src/hazelcore/CursorInfo.re

```diff
@@ -95,7 +95,9 @@ type typed =
   | PatSynthesized(HTyp.t)
   | PatSynKeyword(ExpandingKeyword.t)
   /* cursor in type position */
-  | OnType
+  | TypKeyword(ExpandingKeyword.t)
```

Comment:
what is this for? we don't have any expanding keywords at the type level right?

Reply:
I originally thought `let` and `case` are reserved for both expression position and type position, now I'll delete them.

Reply:
oh I guess we should probably reserve those everywhere for the sake of extraction. you can leave it.